journal.py

```diff
@@ -39,11 +45,26 @@ def write(cls, title=None, text=None, session=None):
         return instance
 
     @classmethod
+    def change(cls, title=None, eid=None, text=None, session=None):
+        if session is None:
+            session = DBSession
+        instance = cls.one(eid)
+        instance.title = title
+        instance.text = text
+        session.add(instance)
+        return instance
```

Comment:
since `eid` is an optional parameter that defaults to `None`, what happens when it is none?  I expect you have tests that demonstrate this?